transformation/src/main/scala/org/broadinstitute/monster/hca/HcaPipeline.scala

```diff
@@ -1,8 +1,11 @@
 package org.broadinstitute.monster.hca
 
+import caseapp.core.help.Help
+import caseapp.core.parser.Parser
 import org.broadinstitute.monster.common.{PipelineBuilder, ScioApp}
 
 /** Entry-point for the HCA pipeline's Docker image. */
-object HcaPipeline extends ScioApp[Args] {
+object HcaPipeline
+    extends ScioApp[Args]()(Parser[Args], Help[Args], postProcess = PostProcess.postProcess) {
```

Comment:
Hmm, maybe a more natural way to express this would've been to make `postProcess` an overrideable `def` in `ScioApp`, just like `pipelineBuilder` is. Making it a class constructor arg seems a bit awkward and the value isn't needed at class construction time.

Reply:
Can do; I think that's definitely cleaner. I'll make that change and update to a newer version of that library then make the change here.